### PR TITLE
including logic

### PR DESCRIPTION
**Pull request by AnaCSilveira**
Including Javascript logic.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    return s1 - s2
+}
+
+function division(d1,d2) {
+    return d1 / d2
+}
+
+function multiplication(m1,m2) {
+    return m1 * m2
+}
```
Comment (assuncaocharles) [inline script.js]: We usually create methods in the infinitive. So, `addition` will be better named as `add` etc...

File: script.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    document.querySelector('.mini-screen').value += r
+    const n1 = parseInt(numberOne)
+    const n2 = parseInt(numberTwo)
+    if (operator == '+') {
+        document.querySelector('.mini-screen').value = (addition(n1,n2))
+    } else if (operator == '-') {
+        document.querySelector('.mini-screen').value = (subtraction(n1,n2))
+    } else if (operator == '/') {
+        document.querySelector('.mini-screen').value = (division(n1,n2))
+    } else { document.querySelector('.mini-screen').value = (multiplication(n1,n2)) }
```
Comment (assuncaocharles) [inline script.js]: You can avoid repeating `document.querySelector('.mini-screen')` by adding it to a variable: 

`const minScreen = document.querySelector('.mini-screen')`

File: script.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (assuncaocharles) [inline script.js:49]: Remember to clean the variables for number and for operation once you have show the result.

**Comment by AnaCSilveira** [comment]
I made the requested adjustments

**Pull request merged**